org/party.go

```diff
@@ -58,7 +58,7 @@ func (p *Party) Calculate() error {
 }
 
 func (p *Party) normalizers() tax.Normalizers {
-	if r := p.RegimeDef(); r != nil {
+	if r := p.RegimeDef(); r != nil && r.Normalizer != nil {
```

Comment:
This feels like it should be handled by a method inside the `tax.Normlizers` as opposed to instantiating an object... I'll have a quick look.

Reply:
Nice one!